### DISCUSSION TITLE
why winget pkg is so big (234MB) when appget was only 2MB

### DISCUSSION THREAD

**Discussion by y1111** [microsoft/winget-cli#4076] [Configuration]
how can the installer here be so big ? whats inside this thing ? it seems like a simple cli program .

**Comment by y1111** [comment]
and looking in the past releases , theres some from may 2023 only 14MB , others 219MB  . what those releases have so different ?

is it the same program so can i download a 14MB one and the same winget will be installed as one of those 200MB ones ?

**Comment by y1111** [comment]
https://github.com/microsoft/winget-cli/releases/tag/v1.5.2201

Aug 9, 2023

seems like that is lthe last version with 15.3MB .

**Reply by y1111** [reply]
https://github.com/microsoft/winget-cli/issues/3652
and as i see here  , this v1.5 doesnt work anymore . in many posts people are pointing that issue to explian that the CDN hardcoded in it is deactivated

**Comment by mdanish-kh** [comment]
See this comment for more info:
- https://github.com/microsoft/winget-cli/discussions/3318#discussioncomment-6113863

**Comment by y1111** [comment]
and also where (in what path) does winget is installed/present in the pc ? because i have it installed but theres no winget anything in system32\  . or it have other name ?

**Reply by denelon** [reply]
WinGet uses App Execution Alias as the mechanism for enabling `winget` to run rather than using a "traditional" path entry. It's part of the MSIX capability set.
